jsx/Form.js

```diff
@@ -1545,7 +1545,6 @@ class CheckboxElement extends React.Component {
     let errorMessage = null;
     let requiredHTML = null;
     let elementClass = this.props.elementClass;
-    let label = null;
```

Comment:
This is never used and causing an error in eslint, which causes `npm run compile` to fail.